infrastructure/variables.tf

```diff
@@ -70,3 +70,9 @@ variable "delete_envelopes_dlq_messages_cron" {
 variable "delete_envelopes_dlq_messages_ttl" {
   default = "72h"
 }
+
+variable "envelopes_queue_max_delivery_count" {
```

Comment:
Maybe this value should just be calculated as the max set at queue level minus some constant, e.g. 5. Otherwise it may be easier to misconfigure it.

Reply:
It is dealt in `main.tf` - whichever is the smallest value  is considered as max delivery count